docs/index.md

```diff
@@ -3,12 +3,62 @@
 page_title: "redpanda Provider"
 subcategory: ""
 description: |-
-  Redpanda Data terraform provider
+  The Redpanda Data Terraform provider is used to manage Redpanda Dedicated and Cloud clusters and Kafka resources within them. To use the provider to connect to a Redpanda Cloud cluster, you need to provide a client_id and client_secret.
+  Example Usage
+  This example demonstrates how to use the Redpanda Terraform provider with various fields populated with dummy values. Ensure you replace these values with your actual data.
+  ```hcl
+  provider "redpanda" {
+    clientid      = "yourclientid"
+    clientsecret  = "yourclientsecret"
```

Comment:
`client_id`, `client_secret`: https://github.com/redpanda-data/terraform-provider-redpanda/blob/main/redpanda/models/redpanda.go#L24-L25

Reply:
i'm not actually sure where tfplugindocs even generated this from

Reply:
ok, i've resolved the faulty index generation